common/display/displayplanemanager.cpp

```diff
@@ -247,12 +248,18 @@ bool DisplayPlaneManager::ValidateLayers(
                   last_plane.SetRotationType(
                       DisplayPlaneState::RotationType::kGPURotation, true);
                 } else {
+                  force_all_layers = false;
                   validate_final_layers = false;
                 }
               }
             }
-
-            break;
+            if (force_all_layers) {
+              ForceGpuForAllLayers(commit_planes, composition, layers,
+                                   mark_later, false);
+              *re_validation_needed = false;
+              *commit_checked = true;
+              return true;
+            }
```

Comment:
you will need a break; here. If we dont return we should make sure we move on to next plane + layer mapping.